File: app/src/main/java/me/magnum/melonds/ui/Theme.java
package me.magnum.melonds.ui;

import androidx.appcompat.app.AppCompatDelegate;

public enum Theme {
    LIGHT(AppCompatDelegate.MODE_NIGHT_NO),
    DARK(AppCompatDelegate.MODE_NIGHT_YES),
    SYSTEM(AppCompatDelegate.MODE_NIGHT_FOLLOW_SYSTEM);

    private int nightMode;

    Theme(int nightMode) {
        this.nightMode = nightMode;
    }

    public int getNightMode() {
        return nightMode;
    }
}

File: app/src/main/java/me/magnum/melonds/ui/Theme.kt
package me.magnum.melonds.ui

import androidx.appcompat.app.AppCompatDelegate

enum class Theme(val nightMode: Int) {
    LIGHT(AppCompatDelegate.MODE_NIGHT_NO),
    DARK(AppCompatDelegate.MODE_NIGHT_YES),
    SYSTEM(AppCompatDelegate.MODE_NIGHT_FOLLOW_SYSTEM);
}